docs/guides/admin/docs/workflowoperationhandlers/speechtotext-attach-woh.md

```diff
@@ -0,0 +1,41 @@
+Speech to Text Attach Workflow Operation
+========================================
+
+ID: `speechtotext-attach`
+
+Description
+-----------
+
+The speech to text attach operation can be used to attach the generated subtitles from a previously run
+[`speechtotext`](speechtotext-woh.md) operation if the process was started asynchronously.
+
+Parameter Table
+---------------
+
+| Configuration Keys       | Required | Example          | Description
+|--------------------------|----------|------------------|-------------
+| target-flavor            | yes      | archive          | Flavor of the produced subtitle
+| target-tags              | no       | captions/source  | Tags for the subtitle file².
```

Comment:
There is a little typo at the end `²`.

As mentioned in my review comment, there could be maybe a note, saying, that the defined tags in the speechtotext operation will be overridden by this target-tags.

Reply:
Fixed with 37642f463882b803fd35cdff1ab280c0e6a9e69b which now clearly states that this option has no effect in asynchronous mode.